README.md

```diff
@@ -1,2 +1,177 @@
-# Hello-Microverse-Project
-The first project
+<a name="readme-top"></a>
+<div align="center">
+  <br/>
+
+  <h1><b>Hello Microvese Project</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Html](#tech-s)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
```

Comment:
[OPTIONAL] Please, I suggest that the word (OPTIONAL) should be removed from the FAQ heading in the table of content and likewise in the content 🙃.